bookkeeper-server/src/main/java/org/apache/bookkeeper/client/EnsemblePlacementPolicy.java

```diff
@@ -50,16 +55,21 @@
      * @param featureProvider feature provider
      * @param statsLogger stats logger
      */
-    public EnsemblePlacementPolicy initialize(ClientConfiguration conf,
+    public default EnsemblePlacementPolicy initialize(ClientConfiguration conf,
                                               Optional<DNSToSwitchMapping> optionalDnsResolver,
                                               HashedWheelTimer hashedWheelTimer,
                                               FeatureProvider featureProvider,
-                                              StatsLogger statsLogger);
+                                              StatsLogger statsLogger) {
+        return initialize(conf);
```

Comment:
@sijie for this comment and the others, the problem is that we must call legacy "initialize" method of 4.4 policies.
So by default we will call the old initialize(conf) which will be noop